client/components/mma/switch/SwitchReview.tsx

```diff
@@ -56,15 +59,58 @@ const whatHappensNextTextCss = css`
 	width: 100%;
 `;
 
+const buttonLayoutCss = css`
+	display: flex;
+	flex-direction: column;
+	margin-top: ${space[5]}px;
+	padding-top: 32px;
+	border-top: 1px solid ${palette.neutral[86]};
+	> * + * {
+		margin-top: ${space[3]}px;
+	}
+	${from.tablet} {
+		flex-direction: row;
+		> * + * {
+			margin-top: 0;
+			margin-left: ${space[3]}px;
+		}
+	}
+`;
+
+const smallPrintCss = css`
+	${textSans.xxsmall()};
+	margin-top: 0;
+	margin-bottom: 0;
+	color: #606060;
+	> a {
+		color: inherit;
+		text-decoration: underline;
+	}
+	& + & {
+		margin-top: ${space[1]}px;
+	}
+`;
+
 interface PreviewResponse {
 	amountPayableToday: number;
 	contributionRefundAmount: number;
 	supporterPlusPurchaseAmount: number;
 }
 
+interface AsyncLoader {
```

Comment:
Not sure about this name - found it a bit confusing when I saw the type use further down. Maybe AsyncLoaderResponse or something.

Reply:
Yeah, it's not a great name actually 😬 I extracted it as an interface as I was going to use it for the second fetch call too, but it's back to only being used by the first one so _could_ return to being done inline? I'll take a look and rename or move it…